surfactant/infoextractors/file_decompression.py

```diff
@@ -87,19 +100,32 @@ def create_extraction(
         # Inherit the context entry install prefix for the extracted files
         install_prefix = current_context.installPrefix
 
-    # Create a temporary directory for extraction
-    temp_folder = create_temp_dir()
-    # Decompress the file
-    entries = decompress(filename, temp_folder)
+    if (
+        software.sha256 in EXTRACT_DIRS
+        and EXTRACT_DIRS[software.sha256]["result"]
+        and os.path.exists(EXTRACT_DIRS[software.sha256]["path"])
+    ):
+        entries = EXTRACT_DIRS[software.sha256]["result"]
+        logger.info(f"Using cached extraction entries for {filename}")
+    else:
+        # Create a temporary directory for extraction
+        temp_folder = create_temp_dir()
```

Comment:
Can we also make a configuration option for the user to set a specific directory to use instead of a temp dir?

It would also probably be good to add a small addition to the docs page for the available settings to list these new options related to extraction/decompression.